examples/countdown.rs

```diff
@@ -3,6 +3,10 @@ use notify_rust::Notification;
 use std::time::Duration;
 
 #[cfg(target_os = "macos")] fn main() { println!("this is a xdg only feature") }
+
```

Comment:
thanks for adapting all these examples. I think this can be done in one place with `any()`